DataFormats/HcalDigi/interface/QIE11DataFrame.h

```diff
@@ -3,13 +3,12 @@
 
 #include "DataFormats/HcalDetId/interface/HcalDetId.h"
 #include "DataFormats/Common/interface/DataFrame.h"
-#include "DataFormats/Common/interface/DataFrameContainer.h"
 #include <ostream>
 
 /** Precision readout digi from QIE11 including TDC information
 
  */
-class QIE11DataFrame : protected edm::DataFrame {
+class QIE11DataFrame {
```

Comment:
These classes were created by @jmmans in 2015 and have not yet been used anywhere, until this PR. This change from inheritance to composition moves to the ECAL style of dataframe classes, which is a lot easier to handle due to some idiosyncrasies when trying to inherit from  edm::DataFrame.